scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,218 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import o3de.command_utils as command_utils
+import o3de.export_project as exp
+import export_utility as eutil
+import o3de.manifest as manifest
+
+from o3de import android, android_support
+
+
+import argparse
+import logging
+import pathlib
+import sys
+
+from typing import List
+
+ASSET_PLATFORM = 'android'
+def export_source_android_project(ctx: exp.O3DEScriptExportContext,
+                           target_android_project_path: pathlib.Path,
+                           seedlist_paths: List[pathlib.Path],
+                           seedfile_paths: List[pathlib.Path],
+                           level_names: List[str],
+                           asset_pack_mode: str,
+                           engine_centric: bool,
+                           should_build_tools: bool,
+                           should_build_all_assets: bool,
+                           build_config: str,
+                           tool_config: str,
+                           tools_build_path: pathlib.Path | None,
+                           max_bundle_size: int,
+                           fail_on_asset_errors: bool,
+                           deploy_to_device: bool,
+                           org_name: str|None,
+                           activity_name:str|None,
+                           logger: logging.Logger|None) -> None:
+    o3de_cli_script_path = ctx.engine_path / ('scripts/o3de' + android_support.O3DE_SCRIPT_EXTENSION)
```

Comment:
f-strings are better here

```suggestion
    o3de_cli_script_path = ctx.engine_path / (f'scripts/o3de{android_support.O3DE_SCRIPT_EXTENSION}')
```